test-executor/src/main/java/com/cognifide/aet/executor/model/TestSuiteRun.java

```diff
@@ -71,7 +71,7 @@ public TestSuiteRun(TestSuiteRun testSuiteRun, String name, String domain, List<
     this.company = testSuiteRun.getCompany();
     this.project = testSuiteRun.getProject();
     this.domain = domain;
-    this.correlationId = testSuiteRun.getCorrelationId();
+    this.correlationId = CorrelationIdGenerator.generateCorrelationId(company, project, name);
```

Comment:
It looks like this class has 2 constructors and now the 2nd constructor could use the 1st one to reduce the amount of code, what do you think? E.g. 2nd constructors could be changed to:
```
public TestSuiteRun(TestSuiteRun testSuiteRun, String name, String domain, List<TestRun> tests) {
    this(name, testSuiteRun.getCompany(), testSuiteRun.getProject(), domain, tests);
  }
```

Reply:
Great suggestion, it seems like a reasonable thing to do.